lib/session.js

```diff
@@ -63,6 +61,8 @@ function signinWithUser(user, req, res, onSuccess) {
         req.session.cookie.maxAge = cookieOpts.maxAge;
       }
       res.cookie('keystone.uid', userToken, cookieOpts);
+      // set user name to cookie for keystone-plugin (chatroom)
+      res.cookie('keystone.user_name', user.name, { maxAge: cookieOpts.maxAge });
```

Comment:
Does keystone.user_name cookie need be set as httpOnly ?

Reply:
No, if we set `httpOnly`, we were not able to retrieve the value on client-side.